include/aws/common/external/cJSON.h

```diff
@@ -25,6 +25,7 @@ THE SOFTWARE.
  *   (1) Address clang-tidy errors by renaming function parameters in a number of places
  *   to match their .c counterparts.
  *   (2) Misc tweaks to unchecked writes to make security static analysis happier
+ *   (3) Comment cJSON_GetErrorPtr and global_error as they are not thread-safe.
```

Comment:
can we just remove the code lines all together instead of commenting them out? looks like a really old issue and dont think it will be fixed any time soon https://github.com/DaveGamble/cJSON/issues/314